elements/pfe-navigation/src/pfe-navigation.html

```diff
@@ -1,4 +1,4 @@
-<div id="pfe-navigation__wrapper" class="pfe-navigation__wrapper" aria-label="Main Navigation">
+<nav id="pfe-navigation__wrapper" class="pfe-navigation__wrapper" aria-label="Main Navigation">
```

Comment:
@wesruv after screen reader testing more I realized that the the nav being there was reading as a secondary nav within the overall pfe-nav which was clunking and kind of confusing and overly verbose to when using a screen reader. It also helps ensure that the screen reader understands that all of the content is a part of the nav not just those menu links. It made the screen reader experience make more sense and easier to understand. It also made sense to me from a semantic point of view since all of the content is a part of the nav, not just those menu items.